src/startupscriptgenerator/src/common/parseUserCommands.go

```diff
@@ -0,0 +1,56 @@
+// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+package common
+
+import (
+	"bufio"
+	"fmt"
+	"os"
+	"strings"
+)
+
+func ParseUserRunCommand(sourcePath string) string {
+
+	appsvcFile, err := os.Open(sourcePath)
+	if err != nil {
+		return ""
+	}
+	defer appsvcFile.Close()
+
+	const runHeading string = "run:" // format of run command- "run: gunicorn myapp.app --workers 5"
+	runCommand := ""
+	var isRunCommandFound bool
+	scanner := bufio.NewScanner(appsvcFile)
+
+	for scanner.Scan() {
+		//fmt.Println(scanner.Text())
+		indexOfRunHeading := strings.Index(scanner.Text(), runHeading)
+
+		if indexOfRunHeading < 0 && strings.Contains(scanner.Text(), ":") && isRunCommandFound {
+			// runCommand already found
+			// not considering any other customized commands
+			break
+		}
+
+		if isRunCommandFound || (indexOfRunHeading >= 0 && len(scanner.Text()) > len(runHeading)) {
+			if isRunCommandFound {
+				runCommand += "\n"
+				runCommand += strings.TrimSpace(scanner.Text())
+			} else {
+				isRunCommandFound = true
+				runCommand += strings.TrimSpace(scanner.Text()[indexOfRunHeading+len(runHeading):]) //gets the run command and trim to get rid of the forward and trailing spaces
+			}
+		}
+	}
+
+	fmt.Println("User provided run command: " + runCommand)
+
+	if err := scanner.Err(); err != nil {
+		return ""
```

Comment:
If an error occurs will the user see any error messages? Here's some recommendations for error handling: https://hussachai.medium.com/error-handling-in-go-a-quick-opinionated-guide-9199dd7c7f76

Reply:
no, the user could not see anything if an error occurs. Now, I added `print` statement to show the error.